.github/workflows/test_cra.yaml

```
@@ -0,0 +1,60 @@
+# on:
```

actionlint

workflow is empty

(syntax-check)